packages/RNTester/js/components/RNTesterListFilters.js

```diff
@@ -15,39 +15,55 @@ const React = require('react');
 const {StyleSheet, Text, TouchableOpacity, View} = require('react-native');
 const filters = ['Basic', 'UI', 'ListViews', 'iOS', 'Android'];
```

Comment:
Adding a newline between imports and local variable declaration.

```suggestion

const filters = ['Basic', 'UI', 'ListViews', 'iOS', 'Android'];
```